Gemfile.lock

```diff
@@ -1,14 +1,14 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.0.5)
```

Comment:
Could you revert your changes on this file? Thanks

Reply:
Yes, sorry about that.